docker/Dockerfile.public_api

```diff
@@ -10,6 +10,7 @@ COPY reactome_analysis_api-*.whl /usr/apps/
 # prepare data structure for the search index
 RUN mkdir -p /data/search_index
 ENV SEARCH_INDEX_PATH /data/search_index
+ENV SEARCH_INDEX_WHITELIST /data/search_index 
```

Comment:
Shouldn't this point to a specific filename? `search_index` is a directory.

Reply:
Yes! changed to whitelist.txt